HznApplication/HznApp.cpp

```diff
@@ -182,10 +183,25 @@ bool SampleLayer::onMouseScroll(Hzn::MouseScrolledEvent& event)
 // ************************************************************************
 
 // *********** EDITOR LAYER **********
-EditorLayer::EditorLayer(const std::string& name) : Layer(name) {}
+EditorLayer::EditorLayer(std::string directoryPath, const std::string& name) : Layer(name) {
+	projectRootFolder = directoryPath;
+	projectPath = "Project(" + projectRootFolder + ")";
```

Comment:
Since you are using `std::filesystem`, have you checked [this](https://en.cppreference.com/w/cpp/filesystem/current_path) out?